portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path

```diff
@@ -990,6 +990,26 @@
 	<td>//li[contains(@class,'nav-item')]//button[contains(@title,'${key_webContentTitle}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>GRAPHQL</td>
+	<td>//button[contains(.,'GraphQL')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>EXECUTE_QUERY</td>
+	<td>//button[contains(@title,'Execute Query (Ctrl-Enter)')]</td>
```

Comment:
maybe we could add this one as well to the GRAPHQL, HEADERS and ADD_HEADER group of generic ones?